synapse/module_api/__init__.py

```diff
@@ -700,6 +715,38 @@ def read_templates(
             (td for td in (self.custom_template_dir, custom_template_directory) if td),
         )
 
+    def ts(self) -> int:
+        """Return a timestamp for the current time, in milliseconds since the epoch"""
+        return self._hs.get_clock().time_msec()
```

Comment:
We've usually been biased towards telling people to just use `time.time()` rather than include a module API that does the exact same thing, so I'd suggest dropping this method.

Reply:
Ah, I hadn't realized, thanks.